File: csharp/Handler.cs
using System;

namespace AwsDotnetCsharp
{
    public class Handler
    {
       public Response Hello(Request request)
       {
           return new Response {
             statusCode = 200,
             body = "hello"
           };
       }
    }

    public class Response
    {
      public int statusCode { get; set;}
      public string body {get; set;}
    }

    public class Request
    {
    }
}

File: java/src/main/java/com/serverless/Handler.java
package com.serverless;

import com.amazonaws.services.lambda.runtime.Context;
import com.amazonaws.services.lambda.runtime.RequestHandler;

import java.util.Map;

public class Handler implements RequestHandler<APIGatewayProxyRequestEvent, APIGatewayProxyResponseEvent> {

    @Override
    public Response handleRequest(APIGatewayProxyRequestEvent input, Context context) {
        return new APIGatewayProxyResponseEvent().withBody("hello").withStatusCode(200);
    }
}